dask/array/core.py

```diff
@@ -2009,6 +2019,83 @@ def from_array(x, chunks, name=None, lock=False, asarray=True, fancy=True,
     return Array(dsk, name, chunks, dtype=x.dtype)
 
 
+def from_zarr(url, component=None, storage_options=None, **kwargs):
```

Comment:
It would be nice to have an option to override the chunks size. Generally have found the chunk size that we choose for storage on disk and what we choose for Dask are different. That might not be everyone's case though. So having it default to the on disk chunk seems sensible. Just the option to override would be good.

Reply:
Should add that choosing a different chunk size to start is way faster than rechunking afterwards. So having the option is pretty important.